### PR TITLE
2017-12-13-aiyaad.md

### PR DESCRIPTION
**Pull request by aiyaad**
Google Code-In is a program that is aimed at encouraging middle-school and high-school students to participate in Open Source. It usually runs for 7-8 weeks during Northern Hemisphere winters. The tasks given in Google Code-In are tasks that would take an experienced well-versed developer 2-3 hours to do so that a student would be able to do it within 5 days. You can apply via Google Melange portal. The applications for Google Code-In are open now. You can choose the organizations, apply for a task and the mentor of a task will choose from the pool of task applicants. If you are a within 13-17 years old and still in school, please apply for Code-In. I had a great experience being a participant there. People are extremely helpful and you will learn a lot.
I registered myself in Google Code-In contest and soon started reading about the organization
and out of so many organizations I like JBoss Community. I love designing things like logos, UI/UX, and a little bit of code. I started with the designing tasks and completed them soon and moved to the coding part where I build a basic iOS app. So far its been a wonderful experience in learning so many things like interacting with mentors and the hurdles you have to cross to complete the tasks,
whenever these hurdles come you experience something new and this experience is kind of a new learning.

I got to know about Open-Source a year before when my brother started working on different open source on Github, I liked what he did and coding kind of attracted me, it looked cool but being an open source developer gives you a lot of experience about a lot of platforms at the same time, I liked the way my brother could pick any project of his own choice to work on. He liked python and knew it. So, he did most of his projects related to python.

So, I guess my open source journey started with this and when I heard about Google Code-In contest, it kind of boosted my energy and I thought I should enter this contest and so I did, and till now it's going well.

The first time I gave up very easily, I always thought that the tasks were too difficult or I didn't understand coding and I was too shy to ask for help on IRC while other students asked about more complex stuff, I thought people will laugh at me.

The second time, I began doing the task and with the grand prize in my mind, I got rid of my shyness, so I began and tried to always be working on a task It was frustrating sometimes but I kept going.

I was doing great, Now I target an average of at least one task per day, and I am going with it. Tasks are a little bit difficult but I don't start with them before I learn the language or design or whatever the task is about, I go through it first and then start working on it.

Till now my Google Code-In experience has been going good and I look for learning more things.

Google Code-In has been a change in my life as I am looking for tasks and eventually new things to learn and so far I am learning and experiencing new things.

Thank You, Google. :)

### CODE REVIEW

**Comment by yashovardhanagrawal** [comment]
you have to update your commit not write it the PR description, go through a git tutorial for this.

File: _posts/2017-12-11-aiyaad.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The tasks given in Google Code-In are tasks that would take 
+an experienced well-versed developer 2-3 hours to do, so that a student would be able to do it within 5 days.
+
+You can apply via Google Melange portal. The applications for Google Code-In are open now. You can choose the organizations, 
+apply for a task and the mentor of a task will choose from the pool of task applicants.
+
+If you are a within 13-17 years old and still in school, please apply for Code-In.
+
+I had a great experience being a participant there. People are extremely helpful and you will learn a lot.
```
Comment (Vedarth) [inline _posts/2017-12-11-aiyaad.md]: You have to make the changes here, i.e. in the file you have added. Kindly go through this https://croisant.net/blog/2008-06-22-git-tip-fix-a-mistake-in-a-previous-commit/.
As an alternative, not recommended though, you can also make another commit to update the file.

Comment (aiyaad) [reply _posts/2017-12-11-aiyaad.md]: I have done changes in the file, can you please check if i have done it right. i even updated the commit

Comment (aiyaad) [reply _posts/2017-12-11-aiyaad.md]: @Vedarth please chk for the changes.

File: _posts/2017-12-11-aiyaad.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+The second time, I began doing the task and with the grand prize in my mind, I got rid of my shyness, so I began and tried to always be working on a task It was frustrating sometimes but i kept going.
+
+I was doing great, Now I target an average of at least one task per day, and i am going with it. Tasks are a little bit difficult but i dont start with them before i learn the language or design or whatever the task is about, i go through it first and then start working on it.
+
+Till now my Google code in experience has been going good and i look for learning more things.
+
+Google Code In has been a change in my life as i am looking for tasks and eventually new things to learn and so far i am leaning and experiencing new things.
+
+Thank You Google. :)
```
Comment (yashovardhanagrawal) [inline _posts/2017-12-11-aiyaad.md]: There are a few language problems, like 'i' is small in many places

you might use 
www.grammarly.com

and improve your blog content in terms of language... 😛

Comment (aiyaad) [reply _posts/2017-12-11-aiyaad.md]: @yashovardhanagrawal Sorry about the grammar. 
I used grammarly and corrected them.
